junseoungbin/likelion-assignment-crud/assignment3_junseoungbin/src/main/java/com/likelion/assignment3_junseoungbin/order/application/OrderService.java

```diff
@@ -0,0 +1,49 @@
+package com.likelion.assignment3_junseoungbin.order.application;
+
+import com.likelion.assignment3_junseoungbin.order.api.dto.request.OrderSaveRequestDto;
+import com.likelion.assignment3_junseoungbin.order.api.dto.response.OrderInfoResponseDto;
+import com.likelion.assignment3_junseoungbin.order.api.dto.response.OrderListResponseDto;
+import com.likelion.assignment3_junseoungbin.order.domain.Order;
+import com.likelion.assignment3_junseoungbin.order.domain.repository.OrderRepository;
+import com.likelion.assignment3_junseoungbin.product.domain.Product;
+import com.likelion.assignment3_junseoungbin.product.domain.repository.ProductRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+@Transactional(readOnly = true)
+public class OrderService {
+    private final OrderRepository orderRepository;
+    private final ProductRepository productRepository;
+
+    @Transactional
+    public void orderSave(OrderSaveRequestDto dto) {
+        Product product = productRepository.findById(dto.productId())
+                .orElseThrow(IllegalArgumentException::new);
```

Comment:
예외 메시지가 없으면 이후에 디버깅 할 때 어떤 오류인지 알기 어려울 수 있어요!
아래처럼 작성하시는 걸 추천드려요~

```suggestion
                .orElseThrow(() -> new IllegalArgumentException("해당 productId에 해당하는 상품을 찾을 수 없습니다."));
```